packages/client/wallets/aa/src/api/CrossmintWalletService.ts

```diff
@@ -58,14 +53,16 @@ export class CrossmintWalletService extends BaseCrossmintService {
         }
     }
 
-    public getPasskeyServerUrl(user: UserParams): string {
-        return (
-            this.crossmintBaseUrl +
-            `/unstable/passkeys/${this.crossmintAPIHeaders["x-api-key"]}/${this.encodeUser(user)}`
+    async fetchNFTs(address: string, chain: EVMBlockchainIncludingTestnet) {
+        return this.fetchCrossmintAPI(
+            `v1-alpha1/wallets/${chain}:${address}/nfts`,
+            { method: "GET" },
+            `Error fetching NFTs for wallet: ${address}`
         );
     }
 
-    private encodeUser(user: UserParams) {
-        return `userId=${user.id}`;
+    public getPasskeyServerUrl(user: UserParams): string {
+        const userParam = `userId=${user.id}`;
+        return this.crossmintBaseUrl + `/unstable/passkeys/${this.crossmintAPIHeaders["x-api-key"]}/${userParam}`;
```

Comment:
just double checking here. The passkeys server won't be auth protected for the first release, right? Because of the hardcoded headers issue?

Reply:
Yeah! Looks like the security review is going to block the release though so we may have time before release though